src/connectors/search-box/connectSearchBox.js

```diff
@@ -131,11 +131,12 @@ export default function connectSearchBox(renderFn, unmountFn) {
 
       dispose({ state }) {
         unmountFn();
+
         return state.setQuery('');
```

Comment:
We don't want to reset the query state to `undefined` on dispose?

Reply:
Yes we do. I didn't changed it (yet) because there are multiple places where this call is neither incorrect or not even implemented e.g `connectAutocomplete`. We'll create another round of PRs to align the calls to `dispose` and `getWidgetSearchParameters`.